Tensile/Source/lib/include/Tensile/Serialization/Predicates.hpp

```diff
@@ -223,6 +223,26 @@ namespace Tensile
                 iot::enumCase(io, value, "gfx1101", AMDGPU::Processor::gfx1101);
                 iot::enumCase(io, value, "gfx1102", AMDGPU::Processor::gfx1102);
             }
+
+            static inline std::unordered_map<std::string, AMDGPU::Processor> lookup = 
```

Comment:
Can we remove the above enumeration?

Reply:
Please fix static analysis errors.

Reply:
Fixed formatting. The other enumeration is likely still used by the YAML codepath, I haven't changed that part of code since it

Reply:
> Fixed formatting. The other enumeration is likely still used by the YAML codepath, I haven't changed that part of code.

Thanks for your update. We can keep the enumeration (I felt it was redundant...).
Why some tests are failing?
Is it caused by your change?